test/blueprints/blueprints.rb

```diff
@@ -296,14 +296,16 @@
 end
 
 TextAnnotation.blueprint do
-  line_start {0}
-  line_end {1}
+  line_start { 0 }
+  line_end { 1 }
+  column_start { 2 }
+  column_end { 3 }
   submission_file
-  is_remark {false}
-  annotation_text {AnnotationText.make(
+  is_remark { false }
+  annotation_text { AnnotationText.make(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.